class-8-js-scopes/yesim-mustafa-js-scope/index.js

```diff
@@ -0,0 +1,85 @@
+// 1- How to make the variable `result` accessible outside the function `add`?
```

Comment:
I didn't find the answers to this activity can you make sure all of them are uncommented and answered

Reply:
Hi Ammar, thanks for the feedback. I revised the JS.